upload.py

```
@@ -185,6 +193,101 @@ def main():
+            print(f"\nSQL statements ({len(results['sql_statements'])}):")
+            for stmt in results['sql_statements']:
+                # Print first line of each statement for readability
+                first_line = stmt.split('\n')[0]
+                print(f"  {first_line}")
+        else:
+            print(f"Synced:  {results['tables_synced']}")
+            print(f"Failed:  {results['tables_failed']}")
+            print(f"BLB-protected (skipped): {results['tables_skipped_blb']}")
+
+            if results['errors']:
+                print(f"\nErrors:")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)